JavaScript/JavaScriptSDK/Telemetry/PageViewPerformance.ts

```diff
@@ -83,7 +83,7 @@ module Microsoft.ApplicationInsights.Telemetry {
                     // some browsers may report individual components incorrectly so that the sum of the parts will be bigger than total PLT
                     // in this case, don't report client performance from this page                    
                     _InternalLogging.throwInternalNonUserActionable(
-                        LoggingSeverity.WARNING,
+                        LoggingSeverity.CRITICAL,
```

Comment:
In case of these errors we won't send client perf. However absence of client perf event may also be caused by lack of API support altogether (safari specific case).

I agree that these messages are not actionable. Also I think we should not show AI (Internal) messages in the console - I'll create a bug.